spec/order_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'order'
+
+describe Order do
+  let(:menu_double) { double :Menu, list: { :shawarma => "£5", :chips => "£3", :salad => "£2", :pepsi => "£1" } }
+  
+  it 'returns the menu on #menu' do
```

Comment:
This test is run in your `menu_spec.rb`. You could look into using your double to return `true` as opposed to the menu. This way you're checking that the Order instance calls this double, without inferring a link between the two classes. (if this didn't make sense, let me know)